webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -325,29 +330,34 @@ export class ReportsComponent implements OnInit, AfterViewInit, OnDestroy {
       return;
     }
 
+    const exportFn = function () {
+      this.exportReports();
+    };
     this.globalActions.setLeftActionBar({
+      exportFn: exportFn.bind(this),
```

Comment:
Nice cleanup here but we could clean it up a bit more this way:
```suggestion
    this.globalActions.setLeftActionBar({
      exportFn: () => this.exportReports(),
```
The arrow function saves us from having to `.bind(this)` the function

Reply:
Oh I tried the arrow function before but it can't call the export service. It looses the context completely

Reply:
That's strange, does it log any specific error? It's working for me

Reply:
👀 okay, I just tried and now it worked... maybe I'm getting crazy but before I got an error about export service not existing and then switched it to regular functions 🤦🏽‍♀️
Thank Mokhtar!!